drivers/sensor/adt7420/adt7420.h

```diff
@@ -10,6 +10,7 @@
 #include <zephyr/types.h>
 #include <device.h>
 #include <gpio.h>
+#include <logging/log.h>
```

Comment:
include this where it is being used, ie. in the .c file where logging functions and macros are being called.